README.md

```diff
@@ -1,2 +1,55 @@
-# orocommerce-shipping-estimator
-A bundle for OroCommerce that adds a shipping estimator component.
+Aligent OroCommerce Shipping Estimator Bundle
+==============================
+This bundle adds the Shipping Estimator functionality on the OroCommerce storefront shopping list.
+
+<img src="src/Aligent/ShippingEstimatorBundle/Resources/doc/img/feature.png" alt="Shipping Estimator functionality">
+
+### Features
+- Enable/disable the Shipping Estimator component on the storefront shopping list
+
+Requirements
+-------------------
+- OroCommerce 4.2
+
+Installation and Usage
+-------------------
+**NOTE: Adjust instructions as needed for your local environment**
+
+### Installation
+Install via Composer
+```shell
+composer require aligent/orocommerce-shipping-estimator
+```
+
+Once installed, run platform update to perform the installation:
+```shell
+php bin/console oro:platform:update --env=prod
+```
+
+
+### Configuration Settings
+
+<img src="src/Aligent/ShippingEstimatorBundle/Resources/doc/img/sytem-config.png" alt="Configuration Options">
```

Comment:
And this should be:
`<img src="src/Resources/doc/img/system-config.png" alt="Configuration Options">`
(note the typo in the filename, it was `sytem` instead of `system`).